### PR TITLE
Update Unicode upgrade instructions

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: lib/elixir/unicode/unicode.ex [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-# 3. Update GraphemeBreakProperty.txt by copying original as is
-# 4. Copy Extended_Pictographic from emoji-data at the end of GraphemeBreakProperty.txt
-# 5. Update GraphemeBreakTest.txt by copying original as is
+# Unicode files can be found in https://www.unicode.org/Public/
+#
+# 1. Replace PropList.txt by copying original
+# 2. Replace SpecialCasing.txt by copying original and removing conditional mappings
+# 3. Replace GraphemeBreakTest.tx by copying original auxiliary/GraphemeBreakTest.txt
+# 4. Replace GraphemeBreakProperty.txt by copying original auxiliary/GraphemeBreakProperty.txt
+# 5. Copy Extended_Pictographic from emoji/emoji-data.txt at the end of GraphemeBreakProperty.txt
```
Comment (josevalim) [inline lib/elixir/unicode/unicode.ex:9]:
```suggestion
# 5. Copy Extended_Pictographic from emoji/emoji-data.txt to the end of GraphemeBreakProperty.txt
```

Comment (eksperimental) [reply lib/elixir/unicode/unicode.ex:9]: actually, it IMO should read:
```
# 5. Append Extended_Pictographic from emoji/emoji-data.txt to the end of GraphemeBreakProperty.txt
```

Comment (josevalim) [reply lib/elixir/unicode/unicode.ex:9]: SGTM!

**Comment by eksperimental** [comment]
done

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**